2024-03-24 05:38:51 ERROR Elements not present after 0.5 seconds on page: https://capital.com/pl/aktualne-cen-towarow?t=most
2024-03-24 05:38:51 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 306, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5638e78caf33 <unknown>
#1 0x5638e75c2ce6 <unknown>
#2 0x5638e760de48 <unknown>
#3 0x5638e760df01 <unknown>
#4 0x5638e76513f4 <unknown>
#5 0x5638e762fedd <unknown>
#6 0x5638e764e899 <unknown>
#7 0x5638e762fc53 <unknown>
#8 0x5638e7600db3 <unknown>
#9 0x5638e760177e <unknown>
#10 0x5638e789086b <unknown>
#11 0x5638e7894885 <unknown>
#12 0x5638e787e181 <unknown>
#13 0x5638e7895412 <unknown>
#14 0x5638e786225f <unknown>
#15 0x5638e78b9528 <unknown>
#16 0x5638e78b9723 <unknown>
#17 0x5638e78ca0e4 <unknown>
#18 0x7f2031a94ac3 <unknown>

2024-03-24 05:38:54 ERROR Elements not present after 0.5 seconds on page: https://capital.com/pl/aktualne-cen-towarow?t=most
2024-03-24 05:38:54 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 306, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5638e78caf33 <unknown>
#1 0x5638e75c2ce6 <unknown>
#2 0x5638e760de48 <unknown>
#3 0x5638e760df01 <unknown>
#4 0x5638e76513f4 <unknown>
#5 0x5638e762fedd <unknown>
#6 0x5638e764e899 <unknown>
#7 0x5638e762fc53 <unknown>
#8 0x5638e7600db3 <unknown>
#9 0x5638e760177e <unknown>
#10 0x5638e789086b <unknown>
#11 0x5638e7894885 <unknown>
#12 0x5638e787e181 <unknown>
#13 0x5638e7895412 <unknown>
#14 0x5638e786225f <unknown>
#15 0x5638e78b9528 <unknown>
#16 0x5638e78b9723 <unknown>
#17 0x5638e78ca0e4 <unknown>
#18 0x7f2031a94ac3 <unknown>

2024-03-24 05:38:58 ERROR Elements not present after 0.5 seconds on page: https://capital.com/pl/aktualne-cen-towarow?t=most
2024-03-24 05:38:58 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 306, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5638e78caf33 <unknown>
#1 0x5638e75c2ce6 <unknown>
#2 0x5638e760de48 <unknown>
#3 0x5638e760df01 <unknown>
#4 0x5638e76513f4 <unknown>
#5 0x5638e762fedd <unknown>
#6 0x5638e764e899 <unknown>
#7 0x5638e762fc53 <unknown>
#8 0x5638e7600db3 <unknown>
#9 0x5638e760177e <unknown>
#10 0x5638e789086b <unknown>
#11 0x5638e7894885 <unknown>
#12 0x5638e787e181 <unknown>
#13 0x5638e7895412 <unknown>
#14 0x5638e786225f <unknown>
#15 0x5638e78b9528 <unknown>
#16 0x5638e78b9723 <unknown>
#17 0x5638e78ca0e4 <unknown>
#18 0x7f2031a94ac3 <unknown>

2024-03-24 05:39:01 ERROR Elements not present after 0.5 seconds on page: https://capital.com/pl/aktualne-cen-towarow?t=most
2024-03-24 05:39:01 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 306, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5638e78caf33 <unknown>
#1 0x5638e75c2ce6 <unknown>
#2 0x5638e760de48 <unknown>
#3 0x5638e760df01 <unknown>
#4 0x5638e76513f4 <unknown>
#5 0x5638e762fedd <unknown>
#6 0x5638e764e899 <unknown>
#7 0x5638e762fc53 <unknown>
#8 0x5638e7600db3 <unknown>
#9 0x5638e760177e <unknown>
#10 0x5638e789086b <unknown>
#11 0x5638e7894885 <unknown>
#12 0x5638e787e181 <unknown>
#13 0x5638e7895412 <unknown>
#14 0x5638e786225f <unknown>
#15 0x5638e78b9528 <unknown>
#16 0x5638e78b9723 <unknown>
#17 0x5638e78ca0e4 <unknown>
#18 0x7f2031a94ac3 <unknown>

2024-03-24 05:39:04 ERROR Elements not present after 0.5 seconds on page: https://capital.com/pl/aktualne-cen-towarow?t=most
2024-03-24 05:39:04 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 306, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5638e78caf33 <unknown>
#1 0x5638e75c2ce6 <unknown>
#2 0x5638e760de48 <unknown>
#3 0x5638e760df01 <unknown>
#4 0x5638e76513f4 <unknown>
#5 0x5638e762fedd <unknown>
#6 0x5638e764e899 <unknown>
#7 0x5638e762fc53 <unknown>
#8 0x5638e7600db3 <unknown>
#9 0x5638e760177e <unknown>
#10 0x5638e789086b <unknown>
#11 0x5638e7894885 <unknown>
#12 0x5638e787e181 <unknown>
#13 0x5638e7895412 <unknown>
#14 0x5638e786225f <unknown>
#15 0x5638e78b9528 <unknown>
#16 0x5638e78b9723 <unknown>
#17 0x5638e78ca0e4 <unknown>
#18 0x7f2031a94ac3 <unknown>

2024-03-24 05:39:07 ERROR Elements not present after 0.5 seconds on page: https://capital.com/pl/aktualne-cen-towarow?t=most
2024-03-24 05:39:07 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 306, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5638e78caf33 <unknown>
#1 0x5638e75c2ce6 <unknown>
#2 0x5638e760de48 <unknown>
#3 0x5638e760df01 <unknown>
#4 0x5638e76513f4 <unknown>
#5 0x5638e762fedd <unknown>
#6 0x5638e764e899 <unknown>
#7 0x5638e762fc53 <unknown>
#8 0x5638e7600db3 <unknown>
#9 0x5638e760177e <unknown>
#10 0x5638e789086b <unknown>
#11 0x5638e7894885 <unknown>
#12 0x5638e787e181 <unknown>
#13 0x5638e7895412 <unknown>
#14 0x5638e786225f <unknown>
#15 0x5638e78b9528 <unknown>
#16 0x5638e78b9723 <unknown>
#17 0x5638e78ca0e4 <unknown>
#18 0x7f2031a94ac3 <unknown>

2024-03-24 05:39:11 ERROR Elements not present after 0.5 seconds on page: https://capital.com/pl/aktualne-cen-towarow?t=most
2024-03-24 05:39:11 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 306, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x5638e78caf33 <unknown>
#1 0x5638e75c2ce6 <unknown>
#2 0x5638e760de48 <unknown>
#3 0x5638e760df01 <unknown>
#4 0x5638e76513f4 <unknown>
#5 0x5638e762fedd <unknown>
#6 0x5638e764e899 <unknown>
#7 0x5638e762fc53 <unknown>
#8 0x5638e7600db3 <unknown>
#9 0x5638e760177e <unknown>
#10 0x5638e789086b <unknown>
#11 0x5638e7894885 <unknown>
#12 0x5638e787e181 <unknown>
#13 0x5638e7895412 <unknown>
#14 0x5638e786225f <unknown>
#15 0x5638e78b9528 <unknown>
#16 0x5638e78b9723 <unknown>
#17 0x5638e78ca0e4 <unknown>
#18 0x7f2031a94ac3 <unknown>